src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/CreateWarehousesActivity/BetterRetailUSA/BRUS-Warehouse5001.json

```
@@ -47,14 +47,14 @@
   "email": null,
   "faxExtension": null,
   "faxNumber": null,
-  "inventoryLocationId": "BRU-5001",
+  "inventoryLocationId": "BRUS-Warehouse5001",
```

is this suppose to be 
```suggestion
  "inventoryLocationId": "BRUS-Warehouse-5001",
```
?